### ISSUE TITLE
Overfitting the validating dataset

### ISSUE THREAD

**Issue by truebelief** [yangyanli/PointCNN#190]
I ran the PointCNN tensorflow code with my own dataset which was split into training, testing and validation datasets. Training and testing accuracies were tracked during training process. After enough training iterations,  both training and testing accuracies are high, close to 0.9. But when the trained model is applied to an independent validation dataset, the accuracy drops to 0.6. This is obviously an overfitting effect. I've used other voxel-based CNN or PointNet which does not encounter such effect. It is not the issue of the dataset.

I suspect the testing dataset actually participated in the training process, although the code turns "is_training" off. If PointCNN uses testing datasets as part of training samples, the high accuracy is not a surprise. I'd appreciate if anyone found the same overfitting problem with an independent validation dataset or just corrected me if wrong.
